src/main/java/com/javadiscord/javabot/commands/user_commands/Regex.java

```diff
@@ -0,0 +1,49 @@
+package com.javadiscord.javabot.commands.user_commands;
+
+import com.javadiscord.javabot.Bot;
+import com.javadiscord.javabot.commands.ResponseException;
+import com.javadiscord.javabot.commands.Responses;
+import com.javadiscord.javabot.commands.SlashCommandHandler;
+import com.javadiscord.javabot.data.properties.config.GuildConfig;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyAction;
+
+import java.util.regex.Pattern;
+
+public class Regex implements SlashCommandHandler {
+
+    GuildConfig config;
+
+    @Override
+    public ReplyAction handle(SlashCommandEvent event) throws ResponseException {
+        config = Bot.config.get(event.getGuild());
+
+        Pattern pattern = Pattern.compile(event.getOption("regex").getAsString());
+        String string = event.getOption("string").getAsString();
```

Comment:
You need to check if the Option is null, only after that you can get it as a String

Reply:
ah yeah duh